apps/vs-code-designer/src/app/commands/workflows/exportLogicApp.ts

```diff
@@ -0,0 +1,432 @@
+    }
+  }
+
+  private async getSubscription(): Promise<any> {
+    const uri = `https://management.azure.com/subscriptions/${this.subscriptionId}/?api-version=2021-04-01`;
+    const options = {
+      method: 'GET',
+      uri,
+      headers: { authorization: this.getAccessToken() },
+      json: true,
+    };
+
+    return requestP(options).catch((error) => {
+      throw new Error(
+        localize('getSubscriptionFailured', 'Failed to get subscription "{0}". {1}', this.subscriptionId, error.message ?? '')
+      );
+    });
+  }
+
+  private async updateParametersAndSettings(
+    output: ConnectionsDeploymentOutput,
+    parametersFile: any,
+    localSettingsFile: any
+  ): Promise<void> {
+    this.addStatus(localize('updateFiles', 'Updating parameters and settings ...'));
+
+    const { value } = output.connections;
+    for (const key of Object.keys(value)) {
+      const item = value[key];
+      parametersFile[item.runtimeUrlParameterKey].value = item.runtimeUrlParameterValue;
+      localSettingsFile.Values[item.authKeyAppsettingKey] = item.authKey;
+    }
+
+    const tenantId = await this.getTenantId();
+    localSettingsFile.Values[workflowTenantIdKey] = tenantId;
+    localSettingsFile.Values[workflowSubscriptionIdKey] = this.subscriptionId;
+    localSettingsFile.Values[workflowResourceGroupNameKey] = this.resourceGroupName;
+    localSettingsFile.Values[workflowLocationKey] = this.location;
+    localSettingsFile.Values[workflowManagementBaseURIKey] = 'https://management.azure.com/';
+
+    writeFileSync(`${this.targetDirectory}/parameters.json`, JSON.stringify(parametersFile, null, 4));
+    writeFileSync(`${this.targetDirectory}/local.settings.json`, JSON.stringify(localSettingsFile, null, 4));
+    this.addStatus(localize('done', 'Done.'));
+  }
+}
+
+export async function exportLogicApp(_context: IActionContext): Promise<void> {
+  const panelName: string = localize('export', 'Export');
+  let accessToken: string;
+  const credentials: ServiceClientCredentials | undefined = await ext.azureAccountTreeItem.getAccountCredentials();
+  const apiVersion = '2021-03-01';
+
+  const dialogOptions: vscode.OpenDialogOptions = {
+    canSelectMany: false,
+    openLabel: 'Select folder',
+    canSelectFiles: false,
+    canSelectFolders: true,
+  };
+
+  const existingPanel: vscode.WebviewPanel | undefined = tryGetWebviewPanel('export', panelName);
+
+  accessToken = await getAuthorizationToken(credentials);
+
+  if (existingPanel) {
+    if (!existingPanel.active) {
+      existingPanel.reveal(vscode.ViewColumn.Active);
+    }
+
+    return;
+  }
+
+  const options: vscode.WebviewOptions & vscode.WebviewPanelOptions = {
+    enableScripts: true,
+    retainContextWhenHidden: true,
+  };
+
+  const panel: vscode.WebviewPanel = vscode.window.createWebviewPanel('ExportLA', `${panelName}`, vscode.ViewColumn.Active, options);
+
+  const indexPath = join(ext.context.extensionPath, 'dist/designer/vs-code-react/index.html');
```

Comment:
You can remove this chunk of code by using the `getWebViewHTML` function, just pass the `vs-code-react` folder name. Here it is an example of how to do it. 
`apps/vs-code-designer/src/app/commands/workflows/openOverview.ts:104`